src/service/riskScoring.js

```diff
@@ -0,0 +1,26 @@
+import getFingerprint from '@endpass/utils/metricsFingerprint';
+import encryptMetric from '@endpass/utils/metricsEncrypt';
+import requestSkipPermission from '@/class/singleton/request/requestSkipPermission';
+import { METRIC_TYPES } from '@/constants';
+
+/**
+ * @param {object[]} entries
+ * @returns {void}
+ */
+const sendUserMetrics = async () => {
+  const fingerprint = await getFingerprint();
```

Comment:
May be, it would be better to wait #326 because it contains full implementation?